lineQueueClient/src/components/Header/Header.js

```diff
@@ -1,49 +1,46 @@
 import React from 'react'
-import {Text, View, StyleSheet, Image} from 'react-native';
+import {Text, View, StyleSheet, Image,} from 'react-native';
 
 /**
  * Renders the app header
  */
 const Header = () => {
   let headerStyle = StyleSheet.create({
+    headerContainer: {
+      flex:1,
+      backgroundColor: '#151515',
```

Comment:
`#151515` should be global variable

Reply:
fixed, defined the color as main in globals.style